### PR TITLE
language updates

### PR DESCRIPTION
**Pull request by jellegerbrandy**
addresses https://daostack.tpondemand.com/RestUI/Board.aspx#page=board/5209716961861964288&appConfig=eyJhY2lkIjoiQjgzMTMzNDczNzlCMUI5QUE0RUE1NUVEOUQyQzdFNkIifQ==&boardPopup=feature/1833/silent

### CODE REVIEW

File: src/components/Dao/ProposalSchemeCard.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ProposalSchemeCard = (props: IProps) => {
           </TrainingTooltip> : headerHtml
         }
         <div>
-          <b>{schemeState.numberOfBoostedProposals}</b> <span>Boosted</span> <b>{schemeState.numberOfPreBoostedProposals}</b> <span>Pending</span> <b>{schemeState.numberOfQueuedProposals}</b> <span>Regular</span>
+          <b>{schemeState.numberOfBoostedProposals}</b> <span>Boosted</span> <b>{schemeState.numberOfPreBoostedProposals}</b> <span>Pending Boosting</span> <b>{schemeState.numberOfQueuedProposals}</b> <span>Regular</span>
```
Comment (tibetsprague) [inline src/components/Dao/ProposalSchemeCard.tsx]: Pending Boosting? really? thats pretty ugly. Pending Boost sounds better to me

File: src/components/Proposal/Create/CreateProposalPage.tsx [hunk 2]
```diff
@@ -44,24 +45,25 @@ class CreateProposalPage extends React.Component<IProps, null> {
     const { daoAvatarAddress } = this.props;
     const scheme = this.props.data;
 
-    const arc = getArc();
-    let schemeName = arc.getContractInfo(scheme.address).name;
-    if (!schemeName) {
-      throw Error(`Unknown Scheme: ${scheme}`);
-    }
+    // const arc = getArc();
```
Comment (tibetsprague) [inline src/components/Proposal/Create/CreateProposalPage.tsx]: we can just remove

File: src/lib/util.ts [hunk 3]
```diff
@@ -236,11 +236,16 @@ export function schemeName(scheme: ISchemeState|IContractInfo, fallback?: string
...
+        // name = `Blockchain Interaction (${contractToCall})`;
+        name = `Blockchain Interaction`;
       }
     } else {
-      name = "Generic Scheme";
+      // this should never happen...
+      name = `Blockchain Interaction`;
     }
+  } else if (scheme.name === "ContributionReward") {
+    name ="Funding and Voting Power"
```
Comment (tibetsprague) [inline src/lib/util.ts]: sorry to say I really don't like this one either 🤷‍♂️ its pretty confusing to me. Vote on Funding would be clearer.

Comment (dOrgJelli) [reply src/lib/util.ts]: What about "Asset Allocation" or "Asset Distribution"?

File: src/components/Dao/DaoHistoryPage.tsx [hunk 4]
```diff
@@ -61,7 +61,7 @@ class DaoHistoryPage extends React.Component<IProps, null> {
                 <tr className={css.proposalHistoryTableHeader}>
                   <th>Proposed by</th>
                   <th>End date</th>
-                  <th>Scheme</th>
+                  <th>Plugin</th>
```
Comment (tibetsprague) [inline src/components/Dao/DaoHistoryPage.tsx]: plugin? really? that makes no sense to me. Scheme makes more sense. who are we targeting with this language?

**Pull request merged**